Assets/Scripts/Boss/BossControl.cs

```diff
@@ -118,14 +118,40 @@ public void BossDeath()
                 if (rb)
                 {
                     rb.isKinematic = false;
-                    rb.AddForce(Vector3.forward * 50 + Vector3.up * 10f, ForceMode.Impulse);
+                    rb.AddForce(Vector3.forward * 500 + Vector3.up * 100f, ForceMode.Impulse);
+                    if (child.transform.position.y < 1.0f) 
+                    {
+                        Debug.Log("child y position is under 1.0f");
+                        Vector3 currentVelocity = rb.velocity;
+
+                        rb.velocity = new Vector3(-currentVelocity.x, -currentVelocity.y, -currentVelocity.z);
```

Comment:
아직 몸뚱이가 땅밑으로 꺼지는 것 같아요.

일단 자연스럽게 가라앉지않게 하려면 y좌표만 음수로 바꿔줘야 할 것 같고, (x, z는 놔두고)

지금은 보스 폭파를 시작하는 시점에만 이 로직이 작동을 해서, 나중에는 몸뚱이의 y좌표가 1 미만이 되더라도 땅에서 튀어올라오지를 않는 상태인거죠.

참고로 저 같으면 동일한 로직을 아래처럼 작성할거 같네요~
```suggestion
                        rb.velocity *= new Vector3(-1, -1, -1);
```

Reply:
넵 폭파 이후에도 추적하도록 하겠습니다!
그런데, rb.velocity *= new Vector3(-1, -1, -1); 이 부분은 유니티에서 지원하지 않는 것 같아서(컴파일 에러가 나네요. rb.velocity.y *= -1 이것도 마찬가지로 에러가 납니다),
일단 기존방식대로 해두었는데, 제가 잘못 알고 있는 거면 말씀해주시면 감사하겠습니닷.

Reply:
아 `*= new Vector3(...)` 이부분은 저도 그냥 될것 같아서 말씀드렸는데 아니면 됐습니다. 무시하셔요 ㅎㅎ